manuscript/manuscript.Rmd

```diff
@@ -112,11 +111,36 @@ Rich, processes-based simulations such as the SORTIE model in forest management
+Such simple, realistic and general models are still the bedrock of most theory and practice today, even when implementation turns to tactical models for quantitative guidance (for instance, the notion of maximum sustainable yield, MSY, in fisheries, or $R_0$ in epidemiology, remain important concepts in management even though they do not exist in typical tactical models).
+These models provide an ideal first benchmark for evaluating the performance of emerging methods of deep RL for several reasons:
+Firstly, for some cases the optimal solution is already known, providing a clear standard-of-comparison to evaluate RL performance. 
+Prior work sometimes overlooks this essential step, assuming that whatever behavior an RL agent produces is sufficiently optimal or otherwise an accurate proxy for behavior arising from animal learning or evolution. 
+As our evaluations will illustrate, such an assumption can be quickly misleading. 
+Second, these models are already widely studied and will be familiar to many readers: @Schaefer1954 is a staple of fisheries management textbooks and practice, with over 2800 citations, while @May1977 has become a canonical model of threshholds and tipping points which still continues to dominate how many ecologists think about these phenomena [@Scheffer2009].  
+Many readers can thus benefit from exisiting knowledge and intutition about the behavior and implications of these models in interpreting the performance of deep RL, something that would not be possible with a more complex model. 
+It would be unwise to consider applications of RL on more complex bespoke models without first analyzing these classic and well-understood cases.
+Third, these models include or can easily be extended to contexts for which the optimal management policy is unknown or inaccessible to classical methods.
+Our implementations of these models in two python modules published to `PyPi` code archive include many such variations which represent open problems for RL.
 We include extensive appendices with carefully annotated code which should allow readers to both reproduce and extend this analysis.
 
+We do recognize the need for more complex, "precise" models to derive quanitatively precise management policy in specific management problems such as fisheries, forestry, or climate change mitigation efforts. 
+The examples here should provide ecologists with the greater background to engage in the collaborative development of such methods, even as they also highlight challenges which ecological problems pose to existing techniques.
+Even the simple models we consider here contain many sources of complexity -- significant stochasticity, non-linear dynamics, hysteresis -- that are likely commonplace to ecological problems but have not been well represtented in RL research to date. 
```

Comment:
I'd consider stochastic and non-linear dynamics to be common in RL, hysteretic dynamics less common.

Reply:
good call, we should re-word that. (though I did try to hedge with 'significant', and of course hysteresis is just one of the many fun things non-linearity can do :-) ). 
Do you think the sentence would read okay if we merely omitted the examples between the `--`?

Reply:
```suggestion
Even the simple models we consider here contain sources of complexity such as hysteresis that are likely commonplace to ecological problems but have not been well represented in RL research to date. 
```

I'm afraid if we don't mention anything it'll come off as vague. Other differentiating sources of complexity I can think of but not necessarily relevant to our environments: huge state-action spaces and poor observability.

Reply:
good call.  I guess the trouble is the sentence here is about "even the simple models we consider" rather than about ecological problems more generally, so we can't really invoke dimensionality etc. I agree that gym_fishing-v1 doesn't really have any model aspects that aren't considered in classic RL, (probably why most algos do fine on the gym itself after a little training), but I think `conservation-v6` really is distinctly harder for them due to the interactions between being (a) stochastic, (b) highly non-linear, including hysteresis, and (c) being non-autonomous system (changing parameter value over the course of the episode).  Perhaps we can be more precise.  How's this?  

> Our first example using a classic optimal control problem for fisheries turns out to be relatively straight-forward for RL, despite some potential pitfalls we explore. Our second example, though still a simple one-dimensional problem, involves the interaction of stochasticity with non-autonomous dynamics (a slowly changing parameter), and highly non-linear dynamics which produce a tipping point and exhibit hysteresis.  This popular ecological model is often used to examine these very phenomena which are thought to be common in ecological systems but have not been well represented in RL research today. These elements can make this problem considerably harder to solve than models in higher dimensions which assume a deterministic and/or autonomous system.  

(maybe too wordy -- suggestions welcome)

Reply:
Looks good to commit, we could maybe make it without the last sentence but this should be easier to decide after seeing it in the manuscript.